src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -73,140 +72,71 @@ TDB_CONTEXT *db = nullptr;
  */
 static int pauseLength = 300;
 
-static FILE *logfile = stderr;
-static int tq_debug_enabled = false;
-
-static void open_log(const char *logfilename)
-{
-    logfile = fopen(logfilename, "a");
-    if ( logfile == NULL ) {
-        perror(logfilename);
-        logfile = stderr;
-    }
-}
-
-static void vlog(const char *level, const char *format, va_list args)
-{
-    time_t now = time(NULL);
-
-    fprintf(logfile, "%ld %s| %s: ", static_cast<long int>(now),
-            program_name, level);
-    vfprintf (logfile, format, args);
-    fflush(logfile);
-}
-
-static void log_debug(const char *format, ...)
-{
-    va_list args;
-
-    if ( tq_debug_enabled ) {
-        va_start (args, format);
-        vlog("DEBUG", format, args);
-        va_end (args);
-    }
-}
-
-static void log_info(const char *format, ...)
-{
-    va_list args;
-
-    va_start (args, format);
-    vlog("INFO", format, args);
-    va_end (args);
-}
-
-static void log_error(const char *format, ...)
-{
-    va_list args;
-
-    va_start (args, format);
-    vlog("ERROR", format, args);
-    va_end (args);
-}
-
-static void log_fatal(const char *format, ...)
+static void init_db(void)
 {
-    va_list args;
+    debugs(DEBUG_SECTION, 2, "opening time quota database \"" << db_path << "\".");
 
-    va_start (args, format);
-    vlog("FATAL", format, args);
-    va_end (args);
-}
+    int dbopts = TDB_CLEAR_IF_FIRST;
 
-static void init_db(void)
-{
-    log_info("opening time quota database \"%s\".\n", db_path);
-    db = tdb_open(db_path, 0, TDB_CLEAR_IF_FIRST, O_CREAT | O_RDWR, 0666);
+    db = tdb_open(db_path, 0, dbopts, O_CREAT | O_RDWR, 0666);
```

Comment:
Undo an unnecessary (and unwanted?) change:

```suggestion
    db = tdb_open(db_path, 0, TDB_CLEAR_IF_FIRST, O_CREAT | O_RDWR, 0666);
```

... and remove dbopts declaration.

Reply:
The change would be wanted, as part of the added option to not wipe the database on squid restarts. Reverting, I'll readd it in the PR to re-add the "preserve" option